sql/moz-fx-data-shared-prod/monitoring_derived/outerbounds_flow_description_v1/query.py

```diff
@@ -2,40 +2,19 @@
 
 """Import outerbounds flow descriptions from Google Sheets."""
 
-import google.auth
-from google.cloud import bigquery
+from pathlib import Path
 
-# Use job config with external table definition
-external_config = bigquery.ExternalConfig("GOOGLE_SHEETS")
-external_config.source_uris = [
-    "https://docs.google.com/spreadsheets/d/1E0kDpHwwtDnkMxAXbeEIzMTajckSHWJ2swF39JdRR1g"
-]
-external_config.schema = [
-    bigquery.SchemaField("flow_name", "STRING"),
-    bigquery.SchemaField("flow_description", "STRING"),
-]
-external_config.options.skip_leading_rows = 1
-job_config = bigquery.QueryJobConfig(
-    table_definitions={"description_sheet": external_config}
-)
+from bigquery_etl.schema import SCHEMA_FILE, Schema
+from bigquery_etl.util import extract_from_query_path
+from bigquery_etl.util.google_sheets import import_google_sheet
 
-# Use credentials that include a google drive scope
-credentials, project = google.auth.default(
-    scopes=[
-        "https://www.googleapis.com/auth/drive",
-        "https://www.googleapis.com/auth/bigquery",
-    ]
-)
-client = bigquery.Client(credentials=credentials, project=project)
+query_path = Path(__file__)
+table_id = ".".join(extract_from_query_path(query_path))
+table_schema = Schema.from_schema_file(query_path.parent / SCHEMA_FILE)
```

Comment:
I wonder if it would have been nicer from usability perspective to pass `query_path.parent` as argument into `import_google_sheet()` which could also be the default value since generally speaking the schema will always be defined as `schema.yaml` in the same directory as the `query.py`?

Reply:
Yes, that should generally be the case when this is used in such `query.py` ETLs, and in an earlier version of the code I actually did have the path being passed in.  However, I ultimately felt it was better to keep the utility function more flexible and not hard-code it rely on specific files existing on disk.  That way it could potentially be used in other scenarios than these ETLs, and if we wanted to add unit tests for this it would be a bit easier to set those tests up.